keep-ui/app/nav.tsx

```diff
@@ -1,7 +1,8 @@
 import Navbar from './navbar';
 import { getServerSession } from '../utils/customAuth';
+import { authOptions } from '../pages/api/auth/[...nextauth]';
 
 export default async function Nav() {
-  const session = await getServerSession();
-  return <Navbar user={session?.user} />;
+  const session = await getServerSession(authOptions);
```

Comment:
getServerSession should go to authUtils and support also single tenant?

Reply:
It already supports single tenant. `getServerSession` is a NextAuth function we override at customAuth